proto/archway/mint/v1/mint.proto

```diff
@@ -4,8 +4,17 @@ package archway.mint.v1;
 option go_package = "github.com/archway-network/archway/x/mint/types";
 
 import "gogoproto/gogo.proto";
+import "google/protobuf/timestamp.proto";
 
 // Params defines the module parameters.
 message Params {
   option (gogoproto.goproto_stringer) = false;
 }
+
+// BlockInfo keeps track of the last block
+message LastBlockInfo {
+  // inflation defines the last block's inflation.
+  string inflation = 1;
```

Comment:
we should add the following options here to make the string a `Dec` on generated code. So we can avoid the validation part:

```
[(gogoproto.customtype) = "github.com/cosmos/cosmos-sdk/types.Dec",
(gogoproto.nullable)   = false,
]
```